cmake/onnxruntime.rc

```diff
@@ -0,0 +1,42 @@
+// This file REQUIRES the following external definitions:
```

Comment:
>// [](start = 0, length = 2)

does this file require copyright notice?

# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.

Reply:
I followed the lead of winml.rc which did not have a copyright notice. Should I add it to both files?

Reply:
Yes, please.